inference-engine/src/cldnn_engine/cldnn_primitives_list.hpp

```diff
@@ -0,0 +1,207 @@
+// REGISTER_FACTORY(v0, Quantize);
+// REGISTER_FACTORY(v0, Select);
+// REGISTER_FACTORY(v0, Subtract);
+// REGISTER_FACTORY(v0, Xor); // Not marked as deprecated yet, but removed from new opsets
+
+// REGISTER_FACTORY(v0, BatchNormInference);
+// REGISTER_FACTORY(v0, Range);
+// REGISTER_FACTORY(v0, RNNCell);
+// REGISTER_FACTORY(v0, ShapeOf);
+// REGISTER_FACTORY(v0, TensorIterator);
+
+// ------------------------------ Supported v0 ops ------------------------------ //
+REGISTER_FACTORY(v1, Add);
+REGISTER_FACTORY(v1, AvgPool);
+REGISTER_FACTORY(v1, BatchToSpace);
+REGISTER_FACTORY(v1, BinaryConvolution);
+REGISTER_FACTORY(v1, Broadcast);
+REGISTER_FACTORY(v1, ConvertLike);
+REGISTER_FACTORY(v1, Convolution);
+REGISTER_FACTORY(v1, ConvolutionBackpropData);
+REGISTER_FACTORY(v1, DeformableConvolution);
+REGISTER_FACTORY(v1, DeformablePSROIPooling);
+REGISTER_FACTORY(v1, Divide);
+REGISTER_FACTORY(v1, Equal);
+REGISTER_FACTORY(v1, FloorMod);
+REGISTER_FACTORY(v1, Gather);
+REGISTER_FACTORY(v1, GatherTree);
+REGISTER_FACTORY(v1, Greater);
+REGISTER_FACTORY(v1, GreaterEqual);
+REGISTER_FACTORY(v1, GroupConvolution);
+REGISTER_FACTORY(v1, GroupConvolutionBackpropData);
+REGISTER_FACTORY(v1, Less);
+REGISTER_FACTORY(v1, LessEqual);
+REGISTER_FACTORY(v1, LogicalAnd);
+REGISTER_FACTORY(v1, LogicalNot);
+REGISTER_FACTORY(v1, LogicalOr);
+REGISTER_FACTORY(v1, LogicalXor);
+REGISTER_FACTORY(v1, MaxPool);
+REGISTER_FACTORY(v1, Maximum);
+REGISTER_FACTORY(v1, Minimum);
+REGISTER_FACTORY(v1, Multiply);
+REGISTER_FACTORY(v1, NotEqual);
+// REGISTER_FACTORY(v1, NonMaxSuppression); Supported via v1 -> v5 internal conversion
+REGISTER_FACTORY(v1, OneHot);
+REGISTER_FACTORY(v1, Pad);
+REGISTER_FACTORY(v1, Power);
+REGISTER_FACTORY(v1, ReduceMax);
+REGISTER_FACTORY(v1, ReduceLogicalAnd);
+REGISTER_FACTORY(v1, ReduceLogicalOr);
+REGISTER_FACTORY(v1, ReduceMean);
+REGISTER_FACTORY(v1, ReduceMin);
+REGISTER_FACTORY(v1, ReduceProd);
+REGISTER_FACTORY(v1, ReduceSum);
+REGISTER_FACTORY(v1, Reshape);
+REGISTER_FACTORY(v1, Subtract);
+REGISTER_FACTORY(v1, SpaceToBatch);
+REGISTER_FACTORY(v1, Softmax);
+REGISTER_FACTORY(v1, StridedSlice);
+REGISTER_FACTORY(v1, Select);
+REGISTER_FACTORY(v1, Split);
+REGISTER_FACTORY(v1, Transpose);
+REGISTER_FACTORY(v1, TopK);
+REGISTER_FACTORY(v1, VariadicSplit);
+
+// ----------------------------- Unsupported v1 ops ----------------------------- //
+// REGISTER_FACTORY(v1, Mod);
+// REGISTER_FACTORY(v1, Reverse);
+
+// ------------------------------ Supported v3 ops ------------------------------ //
+REGISTER_FACTORY(v3, Asinh);
+REGISTER_FACTORY(v3, Acosh);
+REGISTER_FACTORY(v3, Atanh);
+REGISTER_FACTORY(v3, Broadcast);
+REGISTER_FACTORY(v3, EmbeddingBagOffsetsSum);
+REGISTER_FACTORY(v3, EmbeddingBagPackedSum);
+REGISTER_FACTORY(v3, EmbeddingSegmentsSum);
+REGISTER_FACTORY(v3, ExtractImagePatches);
+// REGISTER_FACTORY(v3, NonMaxSuppression); Supported via v3 -> v5 internal conversion
+REGISTER_FACTORY(v3, ScatterUpdate);
```

Comment:
This operation is not supported for now

Reply:
Technically we have primitive and plugin can create it, but the results seem to be wrong. Marked as not implemented temporary.